ci-operator/step-registry/OWNERS

```diff
@@ -0,0 +1,20 @@
+approvers:
```

Comment:
Can you remove these lists? Just the `options:` is what we need

Reply:
updated / thanks